FirebaseAppCheck/Sources/Core/APIService/FIRAppCheckAPIService.m

```diff
@@ -111,18 +111,11 @@ - (instancetype)initWithURLSession:(NSURLSession *)session
 
              [request setValue:self.APIKey forHTTPHeaderField:kAPIKeyHeaderKey];
 
-             NSString *versionHeader =
-                 [NSString stringWithFormat:@"fire-app-check/%@", FIRFirebaseVersion()];
-             [request setValue:versionHeader forHTTPHeaderField:kUserAgentKey];
```

Comment:
Double checking, this version header is intended to be removed, correct?

Reply:
Yes, it's correct.